### PR TITLE
remove template in snippets

### PR DESCRIPTION
**Pull request by morganchen12**
See https://github.com/firebase/snippets-ios/pull/394/files#r1626659811

### CODE REVIEW

File: vertexai/VertexAISnippets/VertexAISnippets.swift [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Snippets {
 
     // Initialize the generative model with a model that supports your use case
     // Gemini 1.5 models are versatile and can be used with all API capabilities
-    let model = vertex.generativeModel(modelName: "{{generic_model_name_initialization}}")
+    let model = vertex.generativeModel(modelName: "gemini-1.5-pro")
```
Comment (andrewheard) [inline vertexai/VertexAISnippets/VertexAISnippets.swift:36]: Please switch this to `gemini-1.5-flash`.

File: vertexai/VertexAISnippets/VertexAISnippets.swift [hunk 2]
```diff
@@ -381,7 +381,7 @@ class Snippets {
     // Initialize the generative model
     // Use a model that supports function calling, like Gemini 1.0 Pro.
```
Comment (andrewheard) [inline vertexai/VertexAISnippets/VertexAISnippets.swift:382]: Would you mind changing that text to "like a Gemini 1.5 model"? It looks like we've gotten out of sync with the latest iteration of the [docs](https://firebase.google.com/docs/vertex-ai/function-calling?platform=ios#specify-function-declaration-during-model-initialization).

File: vertexai/VertexAISnippets/VertexAISnippets.swift [hunk 3]
```diff
@@ -443,7 +443,7 @@ class Snippets {
     // [START function_modes]
     let model = VertexAI.vertexAI().generativeModel(
       // Setting a function calling mode is only available in Gemini 1.5 Pro
-      modelName: "{{generic_model_name_initialization}}",
+      modelName: "gemini-1.5-pro",
```
Comment (andrewheard) [inline vertexai/VertexAISnippets/VertexAISnippets.swift:446]: Note: I think this is the only one of the snippets that needs `gemini-1.5-pro` and the rest can be `gemini-1.5-flash`.

Comment (morganchen12) [reply vertexai/VertexAISnippets/VertexAISnippets.swift:446]: Done.

File: vertexai/VertexAISnippets/VertexAISnippets.swift (hunk 2 above)
Comment (morganchen12) [reply vertexai/VertexAISnippets/VertexAISnippets.swift]: Done.

File: vertexai/VertexAISnippets/VertexAISnippets.swift (hunk 1 above)
Comment (morganchen12) [reply vertexai/VertexAISnippets/VertexAISnippets.swift]: Done.

**Review by andrewheard — APPROVED** [review]
LGTM on visual inspection. Will we get a before/after diff when switching to the includecode? Just want to make sure there weren't any other wording changes.

**Comment by morganchen12** [comment]
Yeah, I'll have it all staged.

**Pull request merged**